class/DatabaseConnection.php

```diff
@@ -0,0 +1,340 @@
+<?php
+
+require_once('config/error_messages.php');
+require_once('config/constants.php');
+require_once('error_helper.php');
+
+/**
+* Super Class
+*
+* @author Smruti Ranjan
+*/
+class DatabaseConnection
+{
+	public static $conn = NULL;
+	public static $connection_obj = NULL;
+
+	/**
+	* Constructor to create a database connection
+	*
+	* @access public
+	* @param  void
+	* @return void[]
+	*/
+	private function __construct($db_param)
+	{
+		self::$conn = mysqli_connect($db_param['hostname'], $db_param['username'], 
+			$db_param['password'], $db_param['database']);
+
+		if(mysqli_connect_errno(self::$conn))
+		{
+			log_error('Creation of databse connection');
```

Comment:
This if case handling Database connection fail. But you logging error as "Connection created".